Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -538,26 +538,24 @@ - (void)tableView:(__attribute__((unused)) UITableView *)tableView
   
   switch(cellKind) {
     case CellKindAgeCheck: {
+      [self.tableView deselectRowAtIndexPath:indexPath animated:YES];
       UITableViewCell *cell = [tableView cellForRowAtIndexPath:indexPath];
-      if (self.selectedAccount.details.userAboveAgeLimit == YES) {
-        [self confirmAgeChange:^(BOOL under13) {
-          if (under13) {
-            cell.accessoryView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"CheckboxOff"]];
```

Comment:
do we still use this `CheckboxOff` and `CheckboxOn` images or can we chuck them?